weather_aplication/SearchViewController.swift

```diff
@@ -0,0 +1,125 @@
+import SnapKit
+import UIKit
+
+struct Location {
+    public var city: String?
+    public var longitude: Double?
+    public var latitude: Double?
+}
```

Comment:
wszystkie pola opcjonalne - czyli może istnieć lokalizacja ani beż nazwy miasta ani współrzędnych?